Asgardia.Constitution.Proposal.txt

```diff
@@ -59,19 +59,19 @@ To ensure the existence and development of Asgardia and humanity’s expansion t
 Article 4. Core Values of Asgardia
 § 1. Asgardia’s Core Values are the common and permanent foundations of the self-identification of Asgardian citizens. These Core Values form the commonality of Asgardian society and lie at the heart of the objectives and activities of Asgardian state bodies, citizens, and associations.
 § 2. The Constitution, laws, and other regulations of Asgardia, as well as the activities of its state bodies, implement and protect Asgardia’s Core Values.
-§ 3. Asgardia’s absolute Prime Directive is humanity reaching out into the Universe, explore and seek out new life, and go where no one has gone before.

```

Comment:
"Absolute" is no longer needed with the use of the phrase "Prime Directive" as this already indicates absolute.